code/modules/ninja/suit/n_suit_verbs/ninja_adrenaline.dm

```diff
@@ -4,11 +4,22 @@
 
 	if(!ninjacost(0,N_ADRENALINE))
 		var/mob/living/carbon/human/H = affecting
-		H.SetUnconscious(0)
+		H.SetSleeping(0)
 		H.SetStun(0)
 		H.SetKnockdown(0)
-		H.adjustStaminaLoss(-75)
+		H.SetUnconscious(0)
+		H.adjustStaminaLoss(-200)
```

Comment:
Stimulants already have heavy stamina regen, -125 would be far less broken.

Reply:
I'd argue a little here and lean towards something like -150.
If you're wounded, even with stimulants you're not going to have a real ton of movespeed to run away with.
Also, to be entirely honest, regular stimulants don't have all that heavy stamina regen either, it's just adjustStaminaLoss(-20*REM, 0) per chem tick.